app/uk/gov/hmrc/apidocumentation/views/AuthorisationUserRestrictedEndpointsView.scala.html

```diff
@@ -706,10 +706,10 @@ <h3 class="govuk-heading-s">Revoking authority</h3>
 
         <h3 class="govuk-heading-s">Agents with multiple accounts</h3>
 
-        <p class="govuk-body">Some agent organisations have more than one Government Gateway account, and their client relationships are spread across those accounts. There are a few ways to handle this:</p>
+        <p class="govuk-body">Some agent organisations have more than one set of HMRC sign in details, and their client relationships are spread across those accounts. There are a few ways to handle this:</p>
```

Comment:
Is `set of` the right terminology? I don't see this mentioned in the ticket.

Reply:
The wording has been approved by Gurmy - I've added a comment to the ticket.